support/index.html

```diff
@@ -3,45 +3,163 @@
+      var code = '';
+      logs.forEach(function (l) {
+        code += '\r\n' + JSON.stringify(l, null, 1);
+      });
+      var html = Prism.highlight(code, Prism.languages.javascript);
+      $('#logger').html(html);
+    }
+    function clearLogs() {
+      logs = [];
+      window.localStorage.removeItem('auth0logs');
+      printLogs();
+    }
+    function subscribeToEvents(instance) {
+      var validEvents = [
+        'show',
+        'hide',
+        'unrecoverable_error',
+        'authenticated',
+        'authorization_error',
+        'hash_parsed',
+        'signin ready',
+        'signup ready',
+        'forgot_password ready',
+        'forgot_password submit',
+        'signin submit',
+        'signup submit',
+        'federated login'
+      ];
+      validEvents.forEach(function (e) {
+        instance.on(e, function () {
+          var args = arguments;
+          if (arguments.length === 1) {
+            args = arguments[0];
+          }
+          logs.push({ event: e, arguments: args });
+          printLogs();
+        });
+      });
+    }
+    $('#btn-clear-log').on('click', clearLogs);
+    $(function () {
+      printLogs();
+    });
+  </script>
   <script>
-    const cid = "BWDP9XS89CJq1w6Nzq7iFOHsTh6ChS2b";
+    const clientId = "BWDP9XS89CJq1w6Nzq7iFOHsTh6ChS2b";
     const domain = "brucke.auth0.com";
-    const options = {
+    const defaultOptions = {
       allowShowPassword: true,
       usernameStyle: 'email',
       defaultDatabaseConnection: 'acme',
       prefill: {
         email: 'johnfoo@gmail.com'
-      }
+      },
+      passwordlessMethod: 'code',
+      allowedConnections: ['sms', 'acme']
     };
-
-    const lock = new Auth0Lock(cid, domain, options);
-    window.lock = lock;
-
-    lock.on("authenticated", function (authResult) {
-      console.log(authResult);
-
-      if (!authResult.accessToken) return;
-
-      lock.getUserInfo(authResult.accessToken, function (error, profile) {
-        console.log(error, profile);
+    function initLock() {
+      var lock = new Auth0Lock(clientId, domain, defaultOptions);
+      window.localStorage.lastUsed = 'lock'
+      subscribeToEvents(lock);
+      return lock;
+    }
+    function initPasswordless() {
+      var lockPasswordless = new Auth0LockPasswordless(clientId, domain, defaultOptions);
+      window.localStorage.lastUsed = 'passwordless'
+      subscribeToEvents(lockPasswordless);
+      return lockPasswordless;
+    }
+    $(function () {
+      $('#btn-show-lock').on('click', function () {
+        clearLogs();
+        var lock = initLock();
+        lock.show({
+          languageDictionary: {
+            title: 'Lock',
+          }
+        });
+      });
+      $('#btn-show-passwordless').on('click', function () {
+        clearLogs();
+        var lockPasswordless = initPasswordless();
+        lockPasswordless.show({
+          languageDictionary: {
+            title: 'Lock + Passwordless',
```

Comment:
Why Lock + Passwordless? Is that true, all lock combinations just by calling `initPasswordless()`?

Reply:
changed